src/smtsolvers/LookaheadSMTSolver.cc

```diff
@@ -239,16 +227,12 @@ std::pair<LookaheadSMTSolver::laresult,Lit> LookaheadSMTSolver::lookaheadLoop()
     bool respect_logic_partitioning_hints = config.respect_logic_partitioning_hints();
     int skipped_vars_due_to_logic = 0;
 
-#ifdef LADEBUG
-    printf("Starting lookahead loop with %d vars\n", nVars());
-#endif
+    Lit best;
```

Comment:
Now there are two `best` declaration in this function. I think the previous version was fine.

Reply:
Fixed